### DISCUSSION TITLE
Reusing vue components in Nova 4

### DISCUSSION THREAD

**Discussion by atmediauk** [laravel/nova-issues#4766] [Ideas & Feature Requests]
- Laravel Version: 9.26.1
- Nova Version: 4.13.0
- PHP Version: 8.1.3
- Database Driver & Version: 10.6.7-MariaDB
- Operating System and Version: macOS 12.3.1
- Browser type and version: Chromium: 104.0.5112.102
- Reproduction Repository: https://github.com/###/###

### Description:
Vue components are not rendered within Laravel Nova 4 blade files. 

### Detailed steps to reproduce the issue on a fresh Nova installation:

In the example below I was testing the <Link> component in the help method for a nova resource field. E.g

**Nova Resource:**
```
            Text::make('Test Field', 'test')
 
                ->help(view('nova.more-info')->render()),
```

**nova.more-info blade file:**
```
 <Link class="text-gray-100" href="#foo"><i class="fas fa-circle-question mr-1"></i> Help</Link>

```

The above displays the unrendered Link tag in the browser with no errors. However the same code/ component renders as expected in a custom tool or field etc (.vue file).

**Comment by crynobone** [comment]
It was never intended to use in such way, marking this as feature request

**Reply by atmediauk** [reply]
> It was never intended to use in such way, marking this as feature request

I see, thanks for the conformation and for marking it as a feature request @crynobone 

Being able to utilise things like ```<Link> <Modal>``` etc components in Novas blade templates would be super useful e.g displaying a popup modal when a fields help icon is clicked is a very clean way to inform a user without being obstructive.

Do you have any ideas/ tips on how to achieve this in the mean time (without creating a custom field for each item)?

Thanks
